source/common/stats/symbol_table_impl.cc

```diff
@@ -215,8 +215,8 @@ Symbol SymbolTableImpl::toSymbol(absl::string_view sv) {
     // a string_view pointing to it in the encode_map_. This allows us to only
     // store the string once. We use unique_ptr so copies are not made as
     // flat_hash_map moves values around.
-    auto str = std::make_unique<std::string>(std::string(sv));
-    auto encode_insert = encode_map_.insert({*str, SharedSymbol(next_symbol_)});
+    InlineStringPtr str = InlineString::create(sv);
```

Comment:
I'm not familiar with this class to say that we should switch to use hand-roll InlineString, considering std::string probably already uses short string optimization.

Reply:
It does provide a slightly space win over std::string in the case where I applied it. I agree it's not a proven win in all cases and I added a caveat.